src/Directory.Build.props

```diff
@@ -1,8 +1,8 @@
 <Project>
   <PropertyGroup>
-    <DebugType>portable</DebugType>
+    <!--<DebugType>portable</DebugType>-->
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="SourceLink.Create.CommandLine" Version="2.1.0" PrivateAssets="all" />
+    <!--<PackageReference Include="SourceLink.Create.CommandLine" Version="2.1.0" PrivateAssets="all" />-->
```

Comment:
I will revert it after finishing this pr and before it get's merged, that was just so I could debug it locally.
Someone else should take care of this, either fix it or just roll https://github.com/fsprojects/Paket/pull/2200/files back.